providers/implementations/kdfs/pkcs12kdf.c

```diff
@@ -151,7 +151,11 @@ static void *kdf_pkcs12_new(void *provctx)
 static void kdf_pkcs12_cleanup(KDF_PKCS12 *ctx)
 {
     ossl_prov_digest_reset(&ctx->digest);
+#ifndef FIPS_MODULE
```

Comment:
This one isn't part of the FIPS provider, so the change here isn't doing anything.
Suggest dropping this one.

Reply:
Yes indeed.